crates/bevy_ui/src/render/mod.rs

```diff
@@ -559,13 +388,6 @@ pub fn extract_uinode_borders(
 
         if let Some(outline) = maybe_outline {
             let outer_distance = uinode.outline_offset() + uinode.outline_width();
-            let outline_radius = border_radius.map(|radius| {
-                if radius > 0. {
-                    radius + outer_distance
-                } else {
-                    0.
-                }
-            });
             let outline_size = uinode.size() + 2. * outer_distance;
```

Comment:
`outline_size` could be moved to a `Node` method too while you're at it.

Reply:
Yeah, called the method `outlined_node_size` as `outline_size` could be confusing.